py/visdom/loggers/keras.py

```
@@ -137,6 +198,28 @@ def on_epoch_end(self, epoch, logs=None):
             warnings.warn(
                 "VisdomKerasLogger failed to log epoch {}: {}".format(epoch, e)
             )
+        if self._params is not None:
+            try:
+                reply = self.viz.log_metrics(logs, step=epoch, env=self.env)
+                self._check_experiment_reply(
+                    reply, "log epoch {} metrics to experiment tracking".format(epoch)
+                )
+            except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)